Page 2.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>J.K. Rowling: Interests and Hobbies </title>
+	    <link rel="stylesheet" type="text/css" href="css/styles.css">
+	    <link rel="stylesheet" type="text/css"
+          href="https://fonts.googleapis.com/css?family=Tangerine">
+          <link rel="stylesheet" type="text/css"
+          href="https://fonts.googleapis.com/css?family=Life+Savers">
+          <link rel="stylesheet" type="text/css"
+          href="https://fonts.googleapis.com/css?family=Eagle+Lake">
+          <link rel="stylesheet" type="text/css"
+          href="https://fonts.googleapis.com/css?family=Megrim">
+          <link rel="stylesheet" type="text/css"
+          href="https://fonts.googleapis.com/css?family=Sorts+Mill+Goudy">
+	</head>
+	<body>
+		<header>
+	      <h1> J.K. Rowling </h1>
+	      <img class=headerimage src="header_image.jpg">
+	      <nav>
+	        <a href="index.html"> Home</a> | <a href="page 2.html"> Interests and Hobbies</a> | 
```

Comment:
@mjhrich this link is not working because of the space in the name.  If generally a separtion of words try using `_` od a `-`